beacon_node/store/src/hot_cold_store.rs

```diff
@@ -573,6 +573,11 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         })
     }
 
+    pub fn has_block(&self, block_root: &Hash256) -> Result<bool, Error> {
+        self.hot_db
+            .key_exists(DBColumn::BeaconBlock.into(), block_root.as_bytes())
+    }
```

Comment:
I think this is the same as [block_exists](https://github.com/sigp/lighthouse/blob/27d468474cdeacd2a8f58b7a8d65c1b5381412ef/beacon_node/store/src/hot_cold_store.rs#L642-L646)